File: stubs/python/connector.py
#!/usr/bin/env python3
import sys, json
for line in sys.stdin:
    line=line.strip()
    if not line: continue
    req=json.loads(line)
    print(json.dumps({"id": req.get("id","0"), "result": {"ok": True}, "error": None}), flush=True)

File: stubs/javascript/connector.js
#!/usr/bin/env node
import readline from 'node:readline';
const rl = readline.createInterface({ input: process.stdin, crlfDelay: Infinity });
for await (const line of rl){
  if(!line.trim()) continue;
  const req = JSON.parse(line);
  process.stdout.write(JSON.stringify({id:req.id||"0", result:{ok:true}, error:null})+"\n");
}
